### DISCUSSION TITLE
Welcome to GMapsFX Discussions!

### DISCUSSION THREAD

**Discussion by dlemmermann** [dlsc-software-consulting-gmbh/GMapsFX#199] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rajghavate98** [comment]
Google Maps API works in Eclipse, Not when Exported as a JAR

**Comment by dlemmermann** [comment]
You are probably using different JDKs and / or JavaFX versions depending on where you run the app. There is an open issue mentioning that GMapsFX stopped working at some point in time after an update to JavaFX. Most likely related to the Webkit version used by JavaFX.
